File: sdk/async-scala/src/main/scala/io/swagger/client/model/OperatorList.scala
package io.swagger.client.model

import org.joda.time.DateTime
import java.util.UUID


case class OperatorList (
  errorcode: Option[String],
errormessage: Option[String],
numberofresults: Option[Integer],
timestamp: Option[String],
results: Option[List[Operator]]
)

File: sdk/javascript-closure-angular/API/Client/OperatorList.js
goog.provide('API.Client.OperatorList');

/**
 * @record
 */
API.Client.OperatorList = function() {}

/**
 * @type {!string}
 * @export
 */
API.Client.OperatorList.prototype.errorcode;

/**
 * @type {!string}
 * @export
 */
API.Client.OperatorList.prototype.errormessage;

/**
 * @type {!number}
 * @export
 */
API.Client.OperatorList.prototype.numberofresults;

/**
 * @type {!string}
 * @export
 */
API.Client.OperatorList.prototype.timestamp;

/**
 * @type {!Array<!API.Client.Operator>}
 * @export
 */
API.Client.OperatorList.prototype.results;

